-------------------------------------------------------------------------------
Test set: com.api.invoice.uts.ApiInvoiceUtsApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.727 s <<< FAILURE! - in com.api.invoice.uts.ApiInvoiceUtsApplicationTests
contextLoads  Time elapsed: 0.016 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext for [WebMergedContextConfiguration@2ba42204 testClass = com.api.invoice.uts.ApiInvoiceUtsApplicationTests, locations = [], classes = [com.api.invoice.uts.ApiInvoiceUtsApplication], contextInitializerClasses = [], activeProfiles = [], propertySourceLocations = [], propertySourceProperties = ["org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true"], contextCustomizers = [org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@352c1b98, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@48d61b48, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@571c5681, org.springframework.boot.test.autoconfigure.actuate.observability.ObservabilityContextCustomizerFactory$DisableObservabilityContextCustomizer@9da1, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@74a6f9c1, org.springframework.boot.test.context.SpringBootTestAnnotation@1b27dd90], resourceBasePath = "src/main/webapp", contextLoader = org.springframework.boot.test.context.SpringBootContextLoader, parent = null]
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Failed to instantiate [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean]: Factory method 'entityManagerFactory' threw exception with message: Failed to get driver instance for jdbcUrl=jdbc:mysql:b3whr0lfnfclciuicxub-mysql.services.clever-cloud.com:3306:clientedb
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean]: Factory method 'entityManagerFactory' threw exception with message: Failed to get driver instance for jdbcUrl=jdbc:mysql:b3whr0lfnfclciuicxub-mysql.services.clever-cloud.com:3306:clientedb
Caused by: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql:b3whr0lfnfclciuicxub-mysql.services.clever-cloud.com:3306:clientedb
Caused by: java.sql.SQLException: No suitable driver

